rider/src/main/kotlin/org/digma/intellij/plugin/rider/protocol/ShowToolWindowHost.kt

```diff
@@ -16,7 +16,7 @@ class ShowToolWindowHost(project: Project) : LifetimedProjectComponent(project)
 
     init {
         model.showToolWindow.advise(project.lifetime){
-            ActivityMonitor.getInstance(project).registerLensClicked()
+            ActivityMonitor.getInstance(project).registerLensClicked("")
```

Comment:
whats the meaning of empty string?

Reply:
I don't remember why there is a call to registerLensClicked, so I remove it